### ISSUE TITLE
Recommended addons: Add Decentraleyes

### ISSUE THREAD

**Issue by nodiscc** [pyllyukko/user.js#150]
Ref https://github.com/pyllyukko/user.js/issues/145#issuecomment-220999839

**Comment by CHEF-KOCH** [comment]
Why not create an pull ?

**Comment by Gitoffthelawn** [comment]
@CHEF-KOCH In another issue (https://github.com/pyllyukko/user.js/issues/145#issuecomment-220848287), you wondered if I had any feedback on Decentraleyes (https://github.com/Synzvato/decentraleyes).  Sorry it's taken me this long to reply.  Been very busy.

In my experience testing Decentraleyes, I found it to work as expected without any issues.  I experienced no reliability issues at all.  I think it is a good tool to recommend.

In it's current implementation, the developer has made a trade-off between extension size and catching more calls to CDN's.  IOW, instead of catching almost all calls and having the extension be 1-2GB, he has chosen to have it catch the *most common calls* and be much smaller in size.  It's a reasonable approach.

There are some ongoing discussions regarding allowing the extension to be extensible to handle even more CDN calls while simultaneously decreasing the extension's size.  I proposed one possible design here: https://github.com/Synzvato/decentraleyes/issues/91#issuecomment-221850897
